R/print.R

```diff
@@ -52,7 +59,14 @@ as.tags.gt_tbl <- function(x, ...) {
   # Attach the dependency to the HTML table
   html_tbl <- htmltools::tagList(
     htmltools::tags$style(htmltools::HTML(css)),
-    tags$div(id = id, style = htmltools::css(`overflow-x` = "auto"), htmltools::HTML(html_table))
+    tags$div(
+      id = id,
+      style = htmltools::css(
+        `overflow-x` = "auto",
+        `overflow-y` = "auto",
```

Comment:
because these are being set locally and how priority works within css, they should be `tab_options` too

Reply:
Good idea. Should we call them `table.overflow.x` and `table.overflow.y`?

Reply:
`container.overflow.x`?

Reply:
Used `container.overflow.*`.